### DISCUSSION TITLE
[Feature] Search: Show related images

### DISCUSSION THREAD

**Discussion by saukkico** [immich-app/immich#12429] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I use the search alot. When I open a photo from the search results, I often would like to see other photos related to that photo. Please add option to see related photos. So for example:
Show photos with same date
Show photos with same location
Show photos with same people 
Show photos taken with same camera
etc.

On mobile those options could be added for example here at the end of the photo info card:
![IMG_1441](https://github.com/user-attachments/assets/a5144e7d-b880-4846-afc6-99ecf4688373)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by iansewell** [comment]
Yes! This would be an amazing.  Should be as simple as jump to the search with the right information fed in.
